src/repr/src/row.rs

```diff
@@ -2841,9 +2851,9 @@ impl SharedRow {
     ///
     /// Panics when the row is already borrowed elsewhere.
     pub fn get() -> Self {
-        let row = Self::SHARED_ROW.with(Rc::clone);
+        let mut row = Self::SHARED_ROW.take().unwrap();
```

Comment:
if I've understood correctly if we unwrap here then it would only fail in the case of a double borrow, correct? should this instead be `expect`?